src/devices/Mcp25xxx/TxBuffer.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Mcp25xxx
+{
+    /// <summary>
+    /// TX Buffers
```

Comment:
can you try to be a bit more explicit on what TX means? I guess it's transmission. So `Transmission buffers` may help in understanding the elements